jane/doc/extensions/_08-miscellaneous-extensions/module-strengthening.md

```diff
@@ -1,3 +1,9 @@
+---
+layout: documentation-page
+collectionName: Miscellaneous Extensions
+title: Module strengthening
```

Comment:
What's our case convention? Headline or not? That is, it feels a little off to have Miscellaneous **E**xtensions and Module **s**trenghtening. I think I favor the latter, uppercasing only the first letter of the heading.

Reply:
It doesn't particularly feel weird to me to pick title case for top-level headings and sentence case for lower level headings. Maybe that's wrong? But in any case I'm happy either way, so I'll adopt your suggestion.